toxcore/group.c

```diff
@@ -1193,6 +1193,15 @@ void g_callback_group_invite(Group_Chats *g_c, g_conference_invite_cb *function)
     g_c->invite_callback = function;
 }
 
+/* Set the callback for group connection.
+ *
+ *  Function(Group_Chats *g_c, void *userdata)
```

Comment:
Let's remove this `Function(...)` stuff. It's documentation that's easy to get out of date and it's not really useful.

Reply:
Done